src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {
```

Comment:
Ball should not extend Lottery
```suggestion
public class Ball {
```